uavcan/time/62804.Synchronization.1.0.uavcan

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
This is pretty much the same as for the old protocol. I've not tested these messages myself. Are they tested to the point that we're confident with publishing them as stabilized definitions? Or are there any weirdness we should discuss first?

Reply:
I am not aware of any issues with these definitions. Maybe @jschall is?